scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -86,7 +86,7 @@ object ScioContext {
   /** Create a new [[ScioContext]] instance for testing. */
   def forTest(): ScioContext = {
```

Comment:
Well class name doesn't really matter in a test so I'd like to keep it this way.